cmd/kwild/server/build.go

```diff
@@ -848,9 +848,15 @@ func buildStatesyncer(d *coreDependencies) *statesync.StateSyncer {
 		failBuild(nil, "failed to configure state syncer, failed to fetch trust options from the remote server.")
 	}
 
+	db, err := d.poolOpener(d.ctx, d.cfg.AppConfig.DBName, 3)
+	if err != nil {
+		failBuild(err, "failed to build db for state syncer")
+	}
+	closer.addCloser(db.Close, "closing state syncer db")
```

Comment:
I think we should just close it here and not wait for closers to do it after kwild shutdown.

Reply:
Oh, I thought it only used the db in the constructor.... dang, nvm